index.html

```diff
@@ -2905,6 +2915,168 @@ <h3>Evidence</h3>
+has been designated to use one or more particular confirmation methods and to
+bind the <a>claims</a> about the <a>subject</a> to one or more of these
+confirmation methods. In this way, an <a>issuer</a> explicitly gives guidance
+and enables a <a>verifier</a> to validate that the <a>holder</a> presenting the
+<a>verifiable credential</a> has proven control of one or more of these
+confirmation methods when the <a>claims</a> bound to the confirmation method
+are presented.
+        </p>
+
+        <p class="note">
+A <a>verifier</a> can decide to accept <a>claims</a> in a <a>verifiable
+credential</a> without validating the confirmation method, or to use a different
+mechanism to validate whether the <a>holder</a> is bound to the presented <a>claims</a>,
+if required. Such decision may prevent impact on the <a>verifier</a>'s
+liability or lack thereof if not specified by other means such
+as a <code>termsOfUse</code> policy.
+        </p>
+
+        <p>
+This specification defines the <code>confirmationMethod</code> <a>property</a>
+for expressing confirmation method information in a
+<code>credentialSubject</code> in a <a>verifiable credential</a>.
+        </p>
+
+        <div class="note">
+          <p>
+For example, an <a>issuer</a> can include a confirmation method based on public
+key cryptography in the <a>verifiable credential</a>. A <a>holder</a> can
+generate and include a <a>proof</a> with a cryptographic signature in the
+<a>verifiable presentation</a>, where the verification key of the cryptographic
+signature is bound to a confirmation method in the embedded <a>verifiable
+credential</a>.
+          </p>
+          <p>
+A <a>verifier</a> can validate that the <a>holder</a> controls
+or has been designated the ability to use the confirmation method
+by verifying the <a>proof</a> of the <a>verifiable
+presentation</a> using the information in the confirmation method. The
+confirmation method can include the verification key, or the type of the
+confirmation method can define that the verification key is to be inferred from other
+<a>properties</a> in the <a>verifiable credential</a>, such as the
+<code>credentialSubject.id</code>.
+          </p>
+        </div>
+
+        <dl>
+          <dt><dfn>confirmationMethod</dfn></dt>
+          <dd>
+          <dd>
+            <p>
+If present, the value of the <code>confirmationMethod</code> <a>property</a> is
+one or more confirmation methods, each providing enough information for a
+<a>verifier</a> to validate that the <a>holder</a> generating a <a>verifiable
+presentation</a> has proven control of a confirmation method bound to one or more
+<a>claims</a> in a <a>verifiable credential</a> in the
+<a>verifiable presentation</a>. It is required that the <a>issuer</a> verifies
+for each <code>confirmationMethod</code> that the <a>holder</a> receiving the
+<code>verifiable credential</code> has proven control of it.
```

Comment:
Might just be me. 

But an issuer is verifiying the holder has control of the confirmationMethod, isn't the proof the thing they verify.

To me, verifying proven control seems odd.

Reply:
Would the following language be better by removing `proven`?

```
It is required that the <a>issuer</a> verifies
for each <code>confirmationMethod</code> that the <a>holder</a> receiving the
<code>verifiable credential</code> has control of it.
```

This sentence was supposed to address a concern that @decentralgabe raised above where the issuer should only include confirmation methods if the holder has control of it.

Reply:
I prefer that. Maybe `control over it`

Reply:
Although thinking about it I am wondering if this is always true. 

Thinking about the case where the credential has multiple claims identifying different subjects, is the suggestion that the issuer would only include confirmationMethods in the claims of the subject who is the holder of the credential.

Or to say another way, can issuers include confirmationMethods for entities who are not the holder at the time of issuance.

Reply:
What about the following...

```
It is required that the <a>issuer</a> verifies
for each <code>confirmationMethod</code> that each <a>subject</a> has control over it.
```

In your marriage certificate, there will be confirmation methods per subject and there should be no problem. 

What do you think?

Reply:
What about 

```
It is required that the <a>issuer</a> verifies the <a>subject</a> has control over each 
<code>confirmationMethod</code> they include in the <a>claims</a> of the <a>verifiable credential</a>s they issue.
```

Again not sure. Wondering about control here still, feels like an artifact of DID Auth based confirmation. Maybe it could be `can satisfy each <code>confirmationMethod</code> ...` 

May just be splitting hairs

Reply:
@wip-abramson can you check if the current language makes sense to you now?

Reply:
Thanks for the feedback, I accepted @TallTed suggestion which you seemed to be fine with too.